### DISCUSSION TITLE
The list of multiple v-select items scroll

### DISCUSSION THREAD

**Discussion by hobuttt** [vuetifyjs/vuetify#14142] [Q&A]
Hi. Why does the list of multiple v-select items scroll down or up when I click on an item? It is not comfortable. Can I turn it off somehow?
https://vuetifyjs.com/en/components/selects/#multiple
Before, like, this was not the case :thinking:

**Comment by wcheek** [comment]
Also seeing this frustrating issue.. The select list always scrolls to the top most selected item.
